Sources/SKSupport/Process+Run.swift

```diff
@@ -82,7 +105,16 @@ extension Process {
         loggingHandler: loggingHandler
       )
     }
-    return process
+    do {
+      return try await Process.launch(
+        arguments: [sh.path, "-c", #"cd $0; "$@""#, workingDirectory.pathString] + arguments,
```

Comment:
Should probably quote `$0` here, also could use `exec` to avoid a fork

```suggestion
        arguments: [sh.path, "-c", #"cd "$0"; exec "$@""#, workingDirectory.pathString] + arguments,
```

Reply:
Great suggestions. Thanks!